### PR TITLE
Change to get all latest running notes in a single call

### PR DESCRIPTION
**Pull request by aanil**

### CODE REVIEW

File: status/flowcells.py [hunk 1]
```diff
@@ -104,13 +105,25 @@ def list_flowcells(self, all=False):
...
 
+        notes = self.application.running_notes_db.view(
+            "latest_note_previews/flowcell",
+            reduce=True,
+            group=True,
+            keys=list(note_keys),
+        )
+        for row in notes:
+            key = row.key
+            if len(row.key.split("_")[0]) > 6:
```
Comment (alneberg) [inline status/flowcells.py:119]: I would add a comment here of what's going on...

Comment (aanil) [reply status/flowcells.py:122]: Done!

**Comment by aanil** [comment]
@alneberg Added something to make sure the latest running note does not take up too much space in tables.

**Comment by alneberg** [comment]
That's probably a good idea. Can you put that on stage as well?

**Comment by aanil** [comment]
Done!

**Comment by alneberg** [comment]
Sorry, forgot about htis one. I am actually not convinced about the `trunc-note-latest` class. It's not really clear when the note has been clipped or not. Can we just skip it for now unless someone really needs it?

**Comment by aanil** [comment]
> Sorry, forgot about this one. I am actually not convinced about the `trunc-note-latest` class. It's not really clear when the note has been clipped or not. Can we just skip it for now unless someone really needs it?

Pär asked for this one, I can confirm with him if he wants it immediately. If he does, I can modify it a bit so that it clearer.

**Comment by alneberg** [comment]
Ok, yes let's talk to Pär 👍

**Review by alneberg — APPROVED** [review]
Thank you! We will have to live without the scrollbars on firefox.

**Pull request merged**